server/controllers/documents.js

```diff
@@ -0,0 +1,154 @@
+import models from '../models';
+import handleResponse from '../helpers/handleResponse';
+import pagination from '../helpers/pagination';
+
+
+/**
+ * define documents controller
+ * @class DocumentsController
+ */
+class DocumentsController {
+
+  /**
+   * @description create document
+   * @param {Object} req - request object from client
+   * @param {Object} res - response object from server
+   * @memberof DocumentsController
+   * @returns {void}
+   */
+  create(req, res) {
+    const authenticadUser = res.locals.decoded;
+    return models.Document.create({
+      title: req.body.title,
+      content: req.body.content,
+      access: req.body.access,
+      userId: authenticadUser.id
+    })
+    .then(document => handleResponse.response(res, 200, { document }))
+    .catch(err => handleResponse.handleError(err, 400, res, 'Validation Error, Invalid input value'));
+  }
+
+  /**
+   * @description get all instance of documents
+   * @param {Object} req - request object from client
+   * @param {Object} res - response object from server
+   * @returns {void}
```

Comment:
`@returns {void}` cannot be correct here as I can clearly see that the function returns something, please fix this everywhere is applies.